docs/node/CATEGORIES/08-account.md

```diff
@@ -176,7 +181,15 @@ enum TransferDomainType {
   DVM = 2,
   /** type for EVM To DVM Token transfer */
   EVM = 3,
-};
+}
+
+enum TransferDomainOptionalKey {
+  SINGLE_KEY_CHECK = 'singlekeycheck',
+}
+
+interface TransferDomainOptionalInfo {
+  singleKeyCheck: true
```

Comment:
```suggestion
  singleKeyCheck: boolean
```
it should be boolean ?

Reply:
true | false is boolean.. specify `true` means "true by default"